### PR TITLE
Extend rbac logs

### PR DESCRIPTION
**Pull request by KSmigielski**

### CODE REVIEW

**Review by matb4r — DISMISSED** [review]

**Review by Ferdudas97 — APPROVED** [review]

File: envoy-control-core/src/main/resources/lua/ingress_rbac_logging.lua [hunk 1]
```diff
@@ -65,58 +73,80 @@ function is_allowed_client(client_name, clients_allowed_to_all_endpoints)
...
+    local path = lua_metadata['request.info.path'] or ''
+    local protocol = handle:connection():ssl() == nil and 'http' or 'https'
+    local method = lua_metadata['request.info.method'] or ''
+    local xff_header = lua_metadata['request.info.xff_header'] or ''
+    local source_ip = string.match(xff_header, '[^,]+$') or ''
+    local request_id = lua_metadata['request.info.request_id'] or ''
+    local status_code = handle:headers():get(':status') or '0'
+    local allowed_client = lua_metadata['request.info.allowed_client'] or false
+    local authority = lua_metadata['request.info.authority'] or ''
+    local lua_destination_authority = lua_metadata['request.info.lua_destination_authority'] or ''
```
Comment (mzawirski) [inline envoy-control-core/src/main/resources/lua/ingress_rbac_logging.lua:106]: how about 'missing' or 'null' instead of empty strings?

**Review by mzawirski — APPROVED** [review]
please create a story for "missing" values in the log (see comment)

File: envoy-control-core/src/main/resources/lua/ingress_rbac_logging.lua (hunk 1 above)
Comment (matb4r) [reply envoy-control-core/src/main/resources/lua/ingress_rbac_logging.lua:106]: Created issue https://github.com/allegro/envoy-control/issues/368

**Comment by matb4r** [comment]
> please create a story for "missing" values in the log (see comment)

https://github.com/allegro/envoy-control/issues/368

**Pull request merged**